### PR TITLE
Get Vlan Id from SAI Vlan Object if bvid present

### PR DESCRIPTION
**Pull request by prsunny**
**- What I did**
Get Vlan Id from SAI_OBJECT_TYPE_VLAN if _bvid_ is present in fdb entry. 

**- How I did it**

**- How to verify it**
Execute fdbshow and it must display the entires

**- Command output**
```
$ fdbshow 
  No.    Vlan  MacAddress         Port
-----  ------  -----------------  ----------
    1    1000  24:8A:07:4C:F5:03  Ethernet12
    2    1000  24:8A:07:4C:F5:18  Ethernet96
    3    1000  24:8A:07:4C:F5:16  Ethernet88
    4    1000  24:8A:07:4C:F5:09  Ethernet36
    5    1000  24:8A:07:4C:F5:15  Ethernet84
```

-->

### CODE REVIEW

**Comment by prsunny** [comment]
This must be merged along with https://github.com/Azure/sonic-py-swsssdk/pull/30

File: scripts/fdbshow [hunk 1]
```diff
@@ -75,8 +75,11 @@ class FdbShow(object):
                 continue
             port_id = self.if_br_oid_map[br_port_id]
             if_name = self.if_oid_map[port_id]
-
-            self.bridge_mac_list.append((int(fdb["vlan"]),) + (fdb["mac"],) + (if_name,))
+            if 'vlan' in fdb:
+                self.bridge_mac_list.append((int(fdb["vlan"]),) + (fdb["mac"],) + (if_name,))
+            elif 'bvid' in fdb:
+                vlan_id = port_util.get_vlan_id_from_bvid(self.db, fdb["bvid"])
+                self.bridge_mac_list.append((int(vlan_id),) + (fdb["mac"],) + (if_name,))
```
Comment (yxieca) [inline scripts/fdbshow]: I have a similar comment here as in snmpagent. Factoring out the common code would be my preference.

**Comment by gitsabari** [comment]
similarly in current code there is no mapping of bridge port OID to name for port channel or other type of interfaces.  for ethernet interfaces COUNTERS_PORT_NAME_MAP has mapping between bridge port OID and name. for LAG is there any mapping table ? 
 if_name = self.if_oid_map[port_id]

same issue is also there for snmpagent.

File: scripts/fdbshow [hunk 2]
```diff
@@ -75,8 +75,11 @@ class FdbShow(object):
                 continue
             port_id = self.if_br_oid_map[br_port_id]
             if_name = self.if_oid_map[port_id]
-
-            self.bridge_mac_list.append((int(fdb["vlan"]),) + (fdb["mac"],) + (if_name,))
+            if 'vlan' in fdb:
+                vlan_id = fdb["vlan"]
+            elif 'bvid' in fdb:
+                vlan_id = port_util.get_vlan_id_from_bvid(self.db, fdb["bvid"])
```
Comment (qiluo-msft) [inline scripts/fdbshow]: >vlan_id [](start = 16, length = 7)

If neither exists, vlan_id will be not defined, do you need protection?

Comment (gitsabari) [reply scripts/fdbshow]: no actually for lag interfaces there are no if_oid_map in this map,  self.if_oid_map[port_id]. that cause it bails out

**Pull request merged**